### PR TITLE
Revise docs and comments of util:get_uuid

### PR DESCRIPTION
**Pull request by LinWencong**
Revise docs and comments of util:get_uuid

### CODE REVIEW

File: paddlets/utils/utils.py [hunk 1]
```diff
@@ -58,13 +58,14 @@ def check_model_fitted(model: Trainable, msg: str = None):
 
 def get_uuid(prefix: str = "", suffix: str = ""):
     """
-    get short UUID
+    Get a random string of 16 characters.
 
     Args:
-        prefix: data to be predicted
+        prefix(str): The prefix of the returned string.
```
Comment (kehuo) [inline paddlets/utils/utils.py:64]: prefix(str) -> prefix(str, optional)

**Review by yangs16 — APPROVED** [review]

File: paddlets/utils/utils.py [hunk 2]
```diff
@@ -58,13 +58,14 @@ def check_model_fitted(model: Trainable, msg: str = None):
...
+        suffix(str): The suffix of the returned string.
```
Comment (kehuo) [inline paddlets/utils/utils.py:65]: suffix(str) -> suffix(str, optional)

**Review by kehuo — CHANGES_REQUESTED** [review]
request changes.

File: paddlets/utils/utils.py [hunk 3]
```diff
@@ -56,13 +57,13 @@ def check_model_fitted(model: Trainable, msg: str = None):
 
     raise_if_not(fitted, msg % {"name": type(model).__name__})
 
-def get_uuid(prefix: str = "", suffix: str = ""):
+def get_uuid(prefix: Optional[str] = "", suffix: Optional[str] = ""):
```
Comment (kehuo) [inline paddlets/utils/utils.py:60]: Only when default value is None, Optional is need. Refers to:
https://docs.python.org/3/library/typing.html#typing.Optional

**Review by kehuo — CHANGES_REQUESTED** [review]
Please remove `Optional`.

**Review by kehuo — APPROVED** [review]
approved.

**Pull request merged**